### PR TITLE
Improve CI

### PR DESCRIPTION
**Pull request by wilwade**
# Problems Fixed

- GitHub wasn't able to require CI statuses pass for matrixes
- Not everything was being checked
- Docker Hub was hitting 429 api limits
- Common changes were not triggering all the tests

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -9,71 +9,125 @@ on:
       - main
 
 jobs:
-  determine-service-matrix:
+  service-matrix:
```
Comment (wilwade) [inline .github/workflows/build.yml:12]: Rename so the lines don't get so long below

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -9,71 +9,125 @@ on:
...
     runs-on: ubuntu-latest
     outputs:
-      services: ${{ steps.determine-matrix.outputs.changes }}
+      changes: ${{ steps.determine-matrix.outputs.changes }}
+      # IF you add a new filter it ALSO needs to be here
+      services: >-
+        ["account", "graph", "content-publishing", "content-watcher"]
```
Comment (wilwade) [inline .github/workflows/build.yml:18]: I tried to find a way not to have to do this, but no luck. This was still the best way.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -9,71 +9,125 @@ on:
...
+        if: ${{ steps.should.outputs.RUN }}
         working-directory: services/${{ matrix.service }}
         run: npm run test
       - name: License Check
+        if: ${{ steps.should.outputs.RUN }}
         working-directory: services/${{ matrix.service }}
         # List all the licenses and error out if it is not one of the supported licenses
         run: npx license-report --fields=name --fields=licenseType | jq 'map(select(.licenseType | IN("MIT", "Apache-2.0", "ISC", "BSD-3-Clause", "BSD-2-Clause", "(Apache-2.0 AND MIT)", "Apache-2.0 OR MIT") | not)) | if length == 0 then halt else halt_error(1) end'
+
+      - name: Build OpenAPI spec
```
Comment (wilwade) [inline .github/workflows/build.yml:91]: Part of the release that broke on v1.0, so this is to keep it from breaking.

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -9,71 +9,125 @@ on:
...
+          platforms: |
+            linux/amd64
+      - name: Set up Docker Buildx
+        if: ${{ steps.should.outputs.RUN }}
+        uses: docker/setup-buildx-action@v3
+        # Use GitHub Container Registry instead due to rate limits
+        with:
+          config-inline: |
+            [registry."ghcr.io"]
+      - name: Cache Docker layers
```
Comment (wilwade) [inline .github/workflows/build.yml:111]: Not really sure this is worth doing, but I did it.

File: .github/workflows/load-tests.yml [hunk 5]
```diff
@@ -9,84 +9,103 @@ on:
       - main
 
 jobs:
-  determine-service-matrix:
+  service-matrix:
```
Comment (wilwade) [inline .github/workflows/load-tests.yml:12]: This is the same set of changes as `build.yml` limited to the matrix changes and no additional jobs

File: .github/workflows/release.yml [hunk 6]
```diff
@@ -53,18 +53,18 @@ jobs:
...
-              echo "::set-output name=SERVICE::${SERVICE}"
-              echo "::set-output name=SERVICE_MATRIX::${SERVICE_MATRIX}"
-              echo "::set-output name=valid::${valid}"
-              echo "::set-output name=test_run::${test_run}"
+              echo "TAG=${TAG}" >> $GITHUB_OUTPUT
+              echo "VERSION=${VERSION_TAG}" >> $GITHUB_OUTPUT
+              echo "SERVICE=${SERVICE}" >> $GITHUB_OUTPUT
+              echo "SERVICE_MATRIX=${SERVICE_MATRIX}" >> $GITHUB_OUTPUT
+              echo "valid=${valid}" >> $GITHUB_OUTPUT
+              echo "test_run=${test_run}" >> $GITHUB_OUTPUT
```
Comment (wilwade) [inline .github/workflows/release.yml:61]: Update to the new version of this

File: services/account/swagger.json
Comment (wilwade) [inline services/account/swagger.json:1]: formatting

File: services/account/apps/api/src/build-openapi.ts [hunk 7]
```diff
@@ -15,7 +15,7 @@ process.env.PROVIDER_ACCOUNT_SEED_PHRASE =
   'offer debate skin describe light badge fish turtle actual inject struggle border';
 process.env.PROVIDER_ID = '0';
 process.env.WEBHOOK_BASE_URL = 'http://127.0.0.1';
-process.env.CAPACITY_LIMIT = '{"type":"amount","value":"1"}';
+process.env.CAPACITY_LIMIT = '{"type":"amount","value":"80"}';
```
Comment (wilwade) [inline services/account/apps/api/src/build-openapi.ts:18]: These are updates to fix the broken swagger builds

**Review by mattheworris — APPROVED** [review]
- Read through changes
Looks good, lgtm!

**Pull request merged**